tests/conftest.py

```diff
@@ -159,9 +189,9 @@ def prepare(
     else:
         file_url = f"s3://luxonis-test-bucket/modelconverter-test/archives/{model_name}.tar.xz"
 
-    if "quant" not in service:
+    if "quant" not in service or "non_quant" in service:
```

Comment:
Can we make this more clear by just using "quant" when we want the quantize flow?

If in some case quant is only option, then we would just defined service with "_quant" in it?

Reply:
Yeah, the tests would use some general refactoring eventually, they're quite convoluted. I'd probably do it all in some separate PR.